src/nodes/chunk_append/planner.c

```diff
@@ -234,7 +247,8 @@ ts_chunk_append_plan_create(PlannerInfo *root, RelOptInfo *rel, CustomPath *path
 															lfirst(lc_childpath),
```

Comment:
I am wondering if the `partial_chunks` parameter for `adjust_childscan` is actually required. Wouldn't it be sufficient in this case to skip the invocation of `adjust_childscan` here?

Reply:
Thank you for reviewing! Hmm most probably, I’m not sure. I ran the test suite and nothing seems to break. The reason I added the parameter instead is because `adjust_childscan` also adjusts the child targetlist but both seem to work as expected.